src/app/Main/Body/QueryPage/QueryPage-Layout.component.tsx

```diff
@@ -112,9 +112,12 @@ export function QueryPageLayoutComponent(props: {
     return (
         <main css={mainStyle}>
             <SubHeaderLayoutComponent
-                exportRowOrStatTable={props.tableExportProps.exportRowOrStatTable}
+                exportRowOrStatTable={
+                    props.tableExportProps.exportRowOrStatTable
+                }
                 isOpen={props.tableExportProps.isOpen}
                 isLoading={props.tableExportProps.isLoading}
+                nrOfIsolates={props.numberOfIsolates.filtered}
```

Comment:
Selecting which numberOfIsolates to use is a task for a container. If this would be a real layout, it would be enforced that the container is responsible for setting the inputs.

Reply:
Done. Commit "Refactoring"